charts/microservice/examples/deploy-nginx.yaml

```diff
@@ -1,5 +1,6 @@
 global:
-  namespace: "demo-dev"
```

Comment:
Do not update this file. Please create new as example values yaml how we can use the affinity and topoloty

Reply:
Have created a affinity.yaml file in examples directory.